feconf.py

```diff
@@ -415,6 +421,7 @@ def get_empty_ratings():
 
 # Unfinished features.
 SHOW_TRAINABLE_UNRESOLVED_ANSWERS = False
+# Synchronize this with base.js
```

Comment:
Reads a bit like a TODO. Maybe: "NOTE TO DEVELOPERS: This should be synchronized with the value in base.js."?